build/dependencies.props

```diff
@@ -1,16 +1,25 @@
 <Project>
   <PropertyGroup>
-    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+    <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <CoreFxVersion>4.3.0</CoreFxVersion>
+    <DapperVersion>1.*</DapperVersion>
+    <InternalAspNetCoreSdkVersion>2.0.0-*</InternalAspNetCoreSdkVersion>
+    <JsonNetVersion>9.0.1</JsonNetVersion>
+    <NETStandardImplicitPackageVersion>$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
     <RuntimeFrameworkVersion>2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
Can be removed. Our other repos have removed this. The SDK now appropriate sets this value to BundledNETCoreAppPackageVersion so that the package version and runtime always line up.